src/renderers/shared/reconciler/__tests__/ReactCompositeComponentError-test.js

```diff
@@ -26,7 +25,9 @@ describe('ReactCompositeComponent-error', function() {
       }
     });
 
+    /*eslint-disable no-new*/
```

Comment:
If we find ourselves needing the rule disabled for other cases let's do that but the single case disable is fine. I don't care how we do it - either is fine by me.